framework/di/Container.php

```diff
@@ -382,11 +382,8 @@ protected function build($class, $params, $config)
         }
 
         $object = $reflection->newInstanceArgs($dependencies);
-        foreach ($config as $name => $value) {
-            $object->$name = $value;
-        }
 
-        return $object;
+        return Yii::configure($object, $config);
```

Comment:
I think container tests are necessary as well with the case as seen in the pull request description.

Reply:
Okey. I'll try

Reply:
I don't agree with this change. Container should not depend on Yii::configure to create object.

Reply:
As for me it's better to leave foreach and handle `Instance` in a special way, as is is done in `\Yii::configure()`